pulp_2to3_migration/app/constants.py

```diff
@@ -8,3 +8,5 @@
 }
 
 NOT_USED = 'Not Used'
+
+BULK_CREATE_BATCH_SIZE = 1000
```

Comment:
I know that this change is not strictly related to the PR, however what do you think about renaming `BULK_CREATE_BATCH_SIZE` to just `BATCH_SIZE` and also update `bulk_update calls to use this constant?

Reply:
Will do.